mlflow/tracking/client.py

```diff
@@ -1527,6 +1527,103 @@ def log_table(
             # Set the tag with the updated list
             self.set_tag(run_id, MLFLOW_LOGGED_ARTIFACTS, json.dumps(current_tag_value))
 
+    @experimental
+    def load_table(
+        self,
+        experiment_id: str,
+        artifact_file: str,
+        run_ids: Optional[List[str]] = None,
+        extra_columns: Optional[List[str]] = None,
```

Comment:
`extra_columns` might conflict with the columns of the downloaded table.

Reply:
Added `extra_` as the prefix and update the documentation to state that.

Reply:
I think we can add a prefix only when conflicts occur. We should also emit a warning.

Reply:
What if we don’t do anything about that?

Just add the column anyway - you’ll have 2 cols with same name, but pandas allows it. Delta might as well


```
In [1]: import pandas as pd
pd.
In [2]: df = pd.DataFrame.from_dict({"a": [1,2,3], "b": [4,5,6]})

In [3]: df.rename(columns={"b": "a"})
Out[3]:
   a  a
0  1  4
1  2  5
2  3  6
```

Reply:
Spoke offline, the conflict approach works fine with Dataframe, but errors out with sparkDataframe with duplicate keys. So we decided to resolve the conflict by appending `_` if there is a conflict. Or else do nothing